loadtest/user/userentity/actions.go

```diff
@@ -156,6 +158,21 @@ func (ue *UserEntity) CreateDirectChannel(otherUserId string) (string, error) {
 
 	return channel.Id, nil
 }
+func (ue *UserEntity) RemoveUserFromChannel(channelId, userId string) error {
+	ok, resp := ue.client.RemoveUserFromChannel(channelId, userId)
+	if !ok {
+		return errors.New("Cant delete the user in the server")
+	}
```

Comment:
I think we could make this function return `(bool, error)` (same as with `Logout()`) and avoid this since it's not really an error case.